prisma/schema.prisma

```diff
@@ -4,7 +4,7 @@ generator client {
 
 datasource db {
   provider = "postgresql"
-  url = env("DATABASE_URL")
+  url = process.env.DATABASE_URL
```

Comment:
this is not wanted right?

Reply:
We were doing the tests, for some reason I forgot to correct the url value. Fixed